packages/strapi-admin/admin/src/containers/HomePage/index.js

```diff
@@ -130,8 +131,18 @@ export class HomePage extends React.PureComponent {
     }
   };
 
-  showFirstBlock = () =>
-    get(this.props.plugins.toJS(), 'content-manager.leftMenuSections.0.links', []).length === 0;
+  showFirstBlock = () => {
+    const { plugins } = this.props;
+    let pluginsObj;
+
+    try {
+      pluginsObj = plugins.toJS();
```

Comment:
Side note this check has been added so we can still switch to the old `AdminPage` container until all features have been merged.

These lines have been added in line files that uses the plugins props.

Reply:
```js
const pluginsObj = plugins.toJS ? plugins.toJS() : plugins;
```